### DISCUSSION TITLE
How to apply an acrylic material to a border ?

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#12209] [Q&A] [answered]
Hey there 👋,

I'm new to Avalonia and while playing around with the controls I discovered that you can apply an acrylic material to the background of a window, which is awesome !

I wonder if it's possible to do the same thing on a border and see the content behind ?

**This is my code in "MainWindow.axaml"** :

```c#
<Grid>
	<Image
		x:Name="_IMAGE"
		Stretch="UniformToFill"/>
	<Border
		x:Name="_BORDER"
		Height="150"
		Width="250"
		VerticalAlignment="Center"
		HorizontalAlignment="Center">
		<Border.Background>
			// ? //
		</Border.Background>
	</Border>
</Grid>
``` 

Thank you for your time ! 😊

**Here are 2 images of what I would like to achieve** :

![Capture d'écran 2023-07-15 153528](https://github.com/AvaloniaUI/Avalonia/assets/139634348/b848583f-ac4c-4dc8-b405-09adf9a3f280)
![Capture d'écran 2023-07-15 153918](https://github.com/AvaloniaUI/Avalonia/assets/139634348/f9c52e0d-9e25-4821-ab41-98577df2965b)

**Comment by timunie** [comment]
looks like this is what you're looking for: https://github.com/AvaloniaUI/Avalonia/discussions/9100#discussioncomment-3800483

for 11.0 the code needs to be adjusted. However, I'm not sure waht exactly needs to be done. Try to figure it out using samples/RenderDemo. 

@maxkatz6 maybe we can update this gist to 11.0? would be really nice.

**Reply by timunie** [reply, marked as answer]
fyi: https://github.com/amwx/FluentAvalonia/discussions/435 this looks pretty close to what you are looking for imo.

**Reply by ghost** [reply]
Hey thank you very much for your answer! Are you the author of Avalonia?
Are you planning to implement an acrylic brush that we could put on a border or a button? That could be something!

Sorry if my knowledges is limited i'm new to Avalonia and i'm even new to Github.

**Reply by timunie** [reply]
I'm a contributor and a part time team member, but I'm mainly here for community support and the maintainer of Avalonia.Samples. Blur effect is somewhat planned, but we need to wait for updates on SkiaSharp first iirc as the current implementation has some pitfalls. So stay tuned and welcome.
